src/aiperf/accuracy/graders/_math_strip.py

```
@@ -0,0 +1,374 @@
+    )
+    string = re.sub(months, "", string, flags=re.IGNORECASE)
+
+    # Add leading 0 to bare ".5" → "0.5"
+    string = string.replace(" .", " 0.")
+    string = string.replace("{.", "{0.")
+
+    # Strip wrapping braces/parens around alphanumeric content
+    if (
+        string.startswith("{")
+        and string.endswith("}")
+        and string.isalnum()
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)